src/app/shared/modules/create-product-modal/create-product-modal.component.html

```diff
@@ -31,6 +31,46 @@ <h1 mat-dialog-title>Create new product</h1>
         {{ getErrorMessage("description") }}
       </mat-error>
     </mat-form-field>
+
+    <div formArrayName="counts">
+      <div
+        class="form-array"
+        *ngFor="let count of counts().controls; let i = index"
+        [formGroupName]="i"
+      >
```

Comment:
Please update formatting here from:

```
     <div
        class="form-array"
        *ngFor="let count of counts().controls; let i = index"
        [formGroupName]="i"
      >   
```

to 


```
     <div *ngFor="let count of counts().controls; let i = index"
          class="form-array"
          [formGroupName]="i">
```

Here and everywhere else(similar cases)